app/templates/inbox/meetings.hbs

```diff
@@ -1,5 +1,5 @@
-<AuDataTable @content={{this.model}} @isLoading={{this.isLoadingModel}} @noDataMessage="Geen zittingen" @sort={{this.sort}} @page={{this.page}} @size={{this.size}} @class="container-flex--contain" as |t|>
-  <t.menu as |menu|>
+<AuDataTable @content={{this.model}} @isLoading={{this.isLoadingModel}} @noDataMessage="Geen zittingen" @sort={{this.sort}} @page={{this.page}} @size={{this.size}} @class="container-flex--contain" as |table|>
```

Comment:
Pro tip, if you start seeing weird rendering errors after adding translations, check that there is no local block variable called `t`. Took me half an hour of debugging before I figured it out :smile:.